### PR TITLE
Mybranch #1736

### PR DESCRIPTION
**Pull request by japhet40**
### Description

Python Time Module: .sleep() #1736

### Type of Change

Fix grammar, spelling, formatting

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@japhet40, It seems you've gone beyond the scope of the Issue and replaced the whole sleep.md file, which is resulting in a conflict so we cannot merge. Can you replace the original sleep.md file and make the edits as requested?

**Comment by SSwiniarski** [comment]
@japhet40 It's been two weeks. Let us know if you're intending to address the issues with this PR. If not we'll have to close this and unassign the issue in question.

**Comment by japhet40** [comment]
> @japhet40, It seems you've gone beyond the scope of the Issue and replaced the whole sleep.md file, which is resulting in a conflict so we cannot merge. Can you replace the original sleep.md file and make the edits as requested?

Is sleep.md suppose to be in this location /docs/content/python/concepts/time-module/terms/sleep ? This is where i have the file with the changes or do i have it all wrong?

**Comment by SSwiniarski** [comment]
@japhet40 the file is in the right place. But the changes you made resulted in a conflict. I can resolve the conflicts and you can apply edits again, if that's all right.

**Comment by japhet40** [comment]
> @japhet40 the file is in the right place. But the changes you made resulted in a conflict. I can resolve the conflicts and you can apply edits again, if that's all right.

ok sounds good if you can resolve the conflict then i will make the edits again once the conflict has been resolved.

**Comment by SSwiniarski** [comment]
@japhet40 I fixed the conflicts and adjusted the PR so it matches the issue being presented. As is it looks ready to merge.

File: media/sleepimage.png
Comment (SSwiniarski) [inline media/sleepimage.png:1]: Remove this unused file.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@japhet40 One more change. Can you remove the extra image file from this PR?

File: media/sleepimage.png
Comment (japhet40) [reply media/sleepimage.png:1]: File removed

**Comment by SSwiniarski** [comment]
@japhet40 the image file is still appearing in this PR. Have you committed the changes to your branch?

**Comment by japhet40** [comment]
> @japhet40 the image file is still appearing in this PR. Have you committed the changes to your branch?

Yes i just committed the changes to my branch .

**Comment by SSwiniarski** [comment]
@japhet40 The file is still appearing in this PR. Are you sure you removed it from your branch?

**Comment by SSwiniarski** [comment]
@japhet40 I went ahead and deleted the file so we can move on to a second review.

**Review by caupolicandiaz — APPROVED** [review]
@japhet40 @SSwiniarski LGTM Just added a couple of minor edits

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@japhet40 Congrats on your entry. Live link is here: https://www.codecademy.com/resources/docs/python/time-module/sleep

**Pull request merged**